style_editor.html

```diff
@@ -287,7 +495,7 @@
 }
 </script>
 
-<script id='hilt_graflex' type='x-shader/x-fragment'>
+<script id="hilt_graflex" type="x-shader/x-fragment">
```

Comment:
Universalizing: double quotes like previous code had throughout.
Plenty more of this eventually; whitespace, indention , clean up in general.
Nothing wrong with perfection?

Reply:
nothing wrong with perfection, but putting everything in one PR is not perfection.